testsuite/fuzzer/fuzz/fuzz_targets/move/aptosvm_publish_and_run.rs

```diff
@@ -420,7 +420,8 @@ fn run_case(mut input: RunnableState) -> Result<(), Corpus> {
     // exec tx
     tdbg!("exec start");
     let mut old_res = None;
-    const N_EXTRA_RERUNS: usize = 3;
+    const N_EXTRA_RERUNS: usize = 0;
```

Comment:
why this change? isn't this used to catch determinism issues?

Reply:
I still think it is not the right place to test for them, execution in the fuzzer is really important and re-executing the same transaction multiple times slows down the process. As I mentioned multiple times the replay tool should check for this or we  run the fuzzer with this feature enabled on another instance. My2cent